tests/mocks/pocketjs.ts

```diff
@@ -115,23 +115,24 @@ const DEFAULT_MOCK_VALUES = {
   ),
 }
 
+type MockRelayResponse = string | RelayResponse | RpcError
+
 export class PocketMock {
   randomizeNodes: boolean
   dispatchers: URL[]
   rpcProvider: HttpRpcProvider
   configuration: Configuration
   nodes: Node[]
-  // relayRequest: string = '{"method":"eth_blockNumber","params":[],"id":1,"jsonrpc":"2.0"}'
-  // relayResponse: string = '{"id":1,"jsonrpc":"2.0","result":"0x1083d57"}'
   fail = false
   rpcMockError = new RpcError('500', 'Mock error')
-  relayRequest = '{"method":"eth_blockNumber","params":[],"id":1,"jsonrpc":"2.0"}'
-  relayResponse: string | RelayResponse | RpcError = '{"id":1,"jsonrpc":"2.0","result":"0x1083d57"}'
+  relayResponse: Record<string, MockRelayResponse | Array<MockRelayResponse>> = {
```

Comment:
to be able to support multiple mocks we can export these to some fixture files and make explicit how we load from fixtures and inject responses into mocks so everyone can instantiate mocks however they like.